numpy/linalg/linalg.py

```diff
@@ -210,7 +210,8 @@ def _assertSquareness(*arrays):
 
 def _assertNdSquareness(*arrays):
     for a in arrays:
-        if max(a.shape[-2:]) != min(a.shape[-2:]):
+        m, n = a.shape[-2:]
```

Comment:
Total nitpick, but I would just write this as
```
if a.shape[-2] != a.shape[-1]:
```
given the error message I think that is clearer than defining `m, n` (and certainly than the min/max thingy there now!).